plat/kvm/include/kvm-arm64/uart.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+#if CONFIG_UART_NS16550
+#include <uart/ns16550.h>
+#endif /* CONFIG UART_NS16550 */
+
+#if CONFIG_UART_PL011
+#include <uart/pl011.h>
+#endif /* CONFIG_UART_PL011 */
+
+static inline void kvm_console_init(void *fdt)
```

Comment:
Why is this in ARM only? Will this be moved to arch agnostic place after plat re-arch?

Reply:
Yes, this will be obsoleted by #986